daemon/images/metrics.go

```diff
@@ -4,11 +4,11 @@ import (
 	metrics "github.com/docker/go-metrics"
 )
 
-var imageActions metrics.LabeledTimer
+var ImageActions metrics.LabeledTimer
```

Comment:
This led me down a bit of dive. I was curious if this variable must be shared, or if it was ok to create a new instance in each package. I noticed this was set through `init()` (did I mention I hate `init()`?), and was even considering; if this must be shared (and now _exported_) would it make sense to use a `sync.Once` and a `ImageActions()` function for this instead of an exported var.

But this is when The funny thing I noticed when looking how it was used elsewhere is that in some places we use `init()` https://github.com/moby/moby/blob/825635a5bfcfc6d860e158dbddea417b323a403c/daemon/events/metrics.go#L10-L11

But in some we just define the package level variable; https://github.com/moby/moby/blob/825635a5bfcfc6d860e158dbddea417b323a403c/daemon/metrics.go#L18-L23


Both of the above initialise a namespace with the same options (`metrics.NewNamespace("engine", "daemon", nil)`), which makes me consider that the variable _may_ not have to be shared (so doesn't have to be a singleton)? 

Do we know if that's correct? (in that case we may not have to export this, and just instance a namespace in each package / implementation, as long as they share the same name and subsystem? If that's _not_ the case, then I'm wondering if some of the existing uses are .. wrong?